src/OrchardCore/OrchardCore.Roles.Abstractions/Extensions/SystemRoleProviderExtensions.cs

```diff
@@ -0,0 +1,13 @@
+namespace OrchardCore.Roles;
+
+public static class SystemRoleProviderExtensions
+{
+    public static bool IsAdminRole(this ISystemRoleProvider systemRoleNameProvider, string name)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(name, nameof(name));
```

Comment:
No need for the second argument

Reply:
What's the role that we need to check?

Reply:
I am not sure what you are asking. I am saying, you don't the second parameter nameof(name)

Reply:
The exception need the argument value and argument name

Reply:
> The exception needs the argument value and argument name

The `paramName` argument has the `CallerArgumentExpressionAtribute` and it takes the argument parameter. So you only need to give it the first parameter (`argument`):

`ThrowIfNullOrEmpty([NotNull] string? argument, [CallerArgumentExpression(nameof(argument))] string? paramName = null)`